portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -104,11 +118,14 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
         // Get the input from the form and log time of comment
         String commentText = request.getParameter(COMMENT_FORM_PARAMETER);
         long timestamp = System.currentTimeMillis();
+        String imageURL = getUploadedFileUrl(request, "image");
+        System.out.println(imageURL);
 
         // Create entity with comment text and timestamp info
         Entity commentEntity = new Entity(COMMENT_KIND);
         commentEntity.setProperty(TEXT_PROPERTY, commentText);
         commentEntity.setProperty(TIMESTAMP_PROPERTY, timestamp);
+        commentEntity.setProperty(IMAGE_URL_PROPERTY, imageURL);
```

Comment:
If the user has not uploaded a file, `imageURL == null`. In that case, would anything break? For instance, will the `Entity` class or `DatastoreService` throw an error?

Reply:
It does not seem to throw an error, and I have code in the JS that just does not show anything if the url is null